test/annexB/built-ins/String/prototype/trimLeft/name.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2017 Valerie Young. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-string.prototype.trimLeft
+description: >
+  String.prototype.trimLeft.name is "trimLeft".
+info: >
+  String.prototype.trimLeft ( )
+
+  17 ECMAScript Standard Built-in Objects:
+    Every built-in Function object, including constructors, that is not
+    identified as an anonymous function has a name property whose value
+    is a String.
+
+    Unless otherwise specified, the name property of a built-in Function
+    object, if it exists, has the attributes { [[Writable]]: false,
+    [[Enumerable]]: false, [[Configurable]]: true }.
+includes: [propertyHelper.js]
+features: [string-trimming]
+---*/
+
+verifyProperty(String.prototype.trimLeft, "name", {
+  value: "trimLeft",
```

Comment:
Is this correct? [According to the proposal’s README](https://github.com/tc39/proposal-string-left-right-trim#naming--aliasing) the `name` should be `trimStart`, although I agree the currently proposed spec text doesn’t make that happen. +@sebmarkbage @evilpie @ljharb

Reply:
It should be `trimStart`. Nice catch.

Reply:
The tests in test/built-ins/String/prototype/* are `trimEnd` and `trimStart`, these are annex B

Reply:
That is irrelevant.

Reply:
That's the confusing part of how this is speced. The property name is what is defined in annex B, but even in annex B, this is the same function instance as trimStart so it has the same `functionInstance.name` field by definition.

Note that it otherwise gets impossible to pass both these name tests and the same value equality test:

https://github.com/tc39/test262/pull/1246/files#diff-c045566571ceee76729db55485a41779

You can't have the same value with two different names.

Reply:
The test is for `trimLeft`, the legacy "web compatibility" method of `String.prototype.`

Reply:
`The function object that is the initial value of  String.prototype.trimLeft is the same function object that is the initial value of String.prototype.trimStart.`

Same as the function in `Array.prototype[Symbol.iterator]` is named `values`, `trimLeft.name` is `"trimStart"`.

thanks for catching this.